src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -50,7 +59,7 @@ const MyChild = () => {
   const meetingManager = useMeetingManager();
   const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
   const [isVoiceFocusEnabled, setIsVoiceFocusEnabled] = useState(false);
-  const [device, setDevice] = useState(meetingManager.selectedAudioInputTransformDevice)
+  const [device, setDevice] = useState(meetingManager.selectedAudioInputDevice);
   const {isVoiceFocusSupported, addVoiceFocus} = useVoiceFocus();
```

Comment:
nit: We add spaces around the curly brackets when using a hook in components. So suggest following the same, here and in the `useVoiceFocus` stories as well:
`const { isVoiceFocusSupported, addVoiceFocus } = useVoiceFocus();`

Reply:
Sure, updated